app/preprints/-components/preprint-institutions/institution-select-list/template.hbs

```diff
@@ -1,30 +1,17 @@
-{{#paginated-list/has-many
-    local-class='institutionList'
-    model=@manager.user
-    relationshipName='institutions'
-    analyticsScope='User Institutions'
-    usePlaceholders=this.usePlaceholders
-    as |list|
-}}
-    <list.item as |institution|>
-        {{#if institution}}
-            <label data-test-institution={{institution.id}} local-class='label'>
-                {{#let (includes institution @manager.affiliatedList) as |isAffiliated|}}
-                    <Input
-                        data-test-institution-button={{concat (if isAffiliated 'remove' 'add') '-' institution.id}}
-                        data-analytics-name={{if isAffiliated 'Remove' 'Add'}}
-                        disabled={{@manager.shouldDisable}}
-                        @type='checkbox'
-                        @checked={{isAffiliated}}
-                        {{on 'change' (fn @manager.toggleInstitution institution)}}
-                    />
-                {{/let}}
+<div local-class='institution-list-container'>
+    {{#each @manager.institutions as |institution index|}}
+        <div local-class='institution-container'>
+            <div local-class='institution-checkbox'>
+                <Input
+                    data-test-institution-input={{index}}
+                    @type='checkbox'
+                    @checked={{institution.isSelected}}
+                    {{on 'change' (fn @manager.toggleInstitution institution)}}
+                />
+            </div>
+            <div local-class='label' data-test-institution-name={{index}}>
                 {{institution.name}}
-            </label>
-        {{else}}
-            <ContentPlaceholders as |placeholder|>
-                {{placeholder.list items=2}}
-            </ContentPlaceholders>
-        {{/if}}
-    </list.item>
-{{/paginated-list/has-many}}
+            </div>
+        </div>
+    {{/each}}
+</div>
```

Comment:
It looks like this component does almost everything it needs to, but I don't see where it calls a `await preprint.save();` to save the changes to the affiliated institutions to the database. Do you need a "Save" (and maybe a "Cancel") button after this list of institutions?

Reply:
Either that or, you can turn `toggleInstitution` into a task that includes `await preprint.save()` at the end